.travis.yml

```diff
@@ -4,17 +4,14 @@ node_js:
 env:
   global:
     - CC_TEST_REPORTER_ID=0be009d55f19435b5d362efac7afcdc532945bbc13527c285135c5e7b84f4e3c
-cache:
-  yarn: true
-  directories:
```

Comment:
I think we still want to cache this?

Reply:
does it by default https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#caching-with-npm